### DISCUSSION TITLE
✅ New cursor movement

### DISCUSSION THREAD

**Discussion by keithellis74** [isontheline/pro.webssh.net#135] [General]
Hi, just a quick note, I love the new implementation of the cursor movement arrow.  So much better. Thank you.

**Comment by isontheline** [comment]
Hi @keithellis74 👋`

Thank you a lot for your feedback!

**Comment by xtchall** [comment]
Is it possible to scroll through the command history using up/down arrow keys at the shell prompt?
And how does one access the up and down arrow keys?

**Comment by keithellis74** [comment]
Yes it is. Arrow keys are on the tool bar


Keith Ellis

**Comment by xtchall** [comment]
Thanks for quick response.

  Had tried a couple times to use that but my fat thumb wasn’t nimble enough - got it to work after some practice with precision moves :)
 
Thanks

**Comment by isontheline** [comment]
If the four "mini" directional keys are not suitable for you, you can also enable the WebSSH "Pad" :
Settings gear > SSH Parameters > Virtual arrow keys

It's bigger.

Could you try?

**Comment by xtchall** [comment]
Oh thanks- I will give that a try.
